categories.md

```diff
@@ -96,12 +96,13 @@ Possibilities:
 3. Overlap with missing names (subset)
 4. Overlap with extra values (actual person names)
 5. Overlap with extra values (noise / not actual names)
-6. No overlap (empty result set)
-7. No overlap (extraneous data)
+6. Some overlap, with both missing names and extra values
+8. No overlap (empty result set)
+9. No overlap (extraneous data)
 
 Questions:
 
-- How to detect automatically whether a name is a person's name or not?
+- How to detect automatically whether a name is a person's name or not? Is this even necessary for the evaluation?
```

Comment:
Maybe not!
I was only thinking of these 2 scenarios:

1. ["Jane", "Doe"] is in the result's author list, but this name is actually the project leader's, or a jury member's - an "honest mistake"
2. ["Public", "Library"] is in the list, the program has picked it up, for example because it was formatted like a possible name (title case, the two words appeared by themselves on a line on the first page...)

Should 2. get a higher penalty then 1. ?

Reply:
2. is a worse answer in the sense that it's not even a name. But 1. is worse in the sense that it's more dangerous - such a mistake might more easily slip through human validation. I would hesitate to give them different penalties, because I can't even tell which one is really worse.

Reply:
If we want to distinguish between names and non-names, it could perhaps be done with a NER tool such as the NER support built into spaCy. The models are language-specific though.

I see this as a rather minor detail in the overall evaluation framework. It's clear the result is wrong, distinguishing between different categories of wrong answers is maybe not that important or could at least be postponed.